{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "python3",
                "purl": "pkg:rpm/vmware/python3?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "python2",
                "purl": "pkg:rpm/vmware/python2?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "glibc",
                "purl": "pkg:rpm/vmware/glibc?distro=photon-3"
            }
        }
    ],
    "id": "PHSA-2019-3.0-0006",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2019-03-06T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-3"
        }
    ],
    "related": [
        "CVE-2019-9169",
        "CVE-2019-9636"
    ]
}